GOVERNANCE.md

```diff
@@ -1,172 +1,73 @@
-- When necessary, will handle human and technical coordination across different working group workstreams.
-
-The working group co-chairs are selected by the members of the working group and are listed in the `CHAIRS` file at the root of this git repository.
-
-## Decision Making
-
-### Deciders
-
-- Repository Maintainers: Decisions that affect only one git repository.
-- Oversight Committee: Decisions that are outside the scope of a single git repository.
-
-### Decision Guidelines
-
-- Decisions that warrant wider input should be made public by using the below guidelines in combination with the Proposal Process below.
-- Whether or not wider input is required, the GitOps Working Group community believes that the best decisions are reached through Consensus <https://en.wikipedia.org/wiki/Consensus_decision-making>.
-- Most decisions start by seeking Lazy Consensus <https://communitymgt.wikia.com/wiki/Lazy_consensus>.
-- If an objection is raised through the Lazy Consensus process, Deciders work together to seek an agreeable solution.
-- If Consensus can not be reached, but a decision must be made, the next step is try to attempt to agree that a vote should be called.
-  This is important, as it gives dissenting views a chance to request more information or raise further points.
-  If Deciders are the Oversight Committee, part of that responsibility is the final point of escalation, so agreeing to a vote is assumed if timeline doesn't allow the consensus process to continue.
-- If Deciders are Repository Maintainers, and they can't agree on calling a vote, they may escalate to the Oversight Committee.
-  This should only be done at this stage if:
-  1. An unmovable deadline is threatened by continuing the Consensus process; or
-  2. A Decider feels there is unreasonable blocking of both reaching Consensus and agreeing to a vote.
-      This should be rare, due to the social cost of discontinuing the Consensus process for this reason.
-      Most decisions should wait for the above process to take its course.
-- If Deciders agree to a vote, the default is a Simple Majority.
-- However, there are cases that require stronger voting – Supermajority or Unanimity – specified below:
-
-### Simple Majority Decisions
-
-If a vote is called, the default is a Simple Majority Vote <https://en.wikipedia.org/wiki/Majority>.
+- Engage deeply in the work being done by the working group - that is, the role is not purely administrative.
+  Chairs are, however not responsible for performing all of the work - this is shared across the working group members and expected from those who have volunteered onto various workstreams.
+- When necessary, handle human and technical coordination across different working group workstreams
```

Comment:
What do y'all think about also considering the definition of new initiatives as well as the promotion of open involvement as responsibilities for the chairs?

Reply:
> What do y'all think about also considering the definition of new initiatives 

I was thinking that's covered by this? WDYT
```
- Schedule discussion of proposals that have been submitted
- Partner with other WG members to establish a roadmap and manage ongoing projects
- Ask for new proposals to be made to address an identified needs
```

> as well as the promotion of open involvement as responsibilities for the chairs?

Maybe "Ensure the promotion of open involvement"? This way, it's not the sole responsibility of chairs to promote this, but only to ensure that it is in fact being promoted.

On this note, I had set up a "Communications" team in this GitHub org as an area to ask other interested people for help. For consistent messaging, I do think we need a dedicated team for this – however small – with good internal communication and clear processes. So yes, sgtm!

Reply:
@murillodigital  Added your second suggestion with commit co-authorship 👍  LMK if you disagree with the first point